src/prelude.rs

```diff
@@ -3,10 +3,16 @@
 pub use crate::chunks::metadata::Metadata;
 
 pub use crate::chunks::storage::{
-    MmapArray, NdArray, QuantizedArray, Storage, StorageView, StorageViewWrap, StorageWrap,
+    MmapArray, MmapQuantizedArray, NdArray, QuantizedArray, Storage, StorageView, StorageViewWrap,
+    StorageWrap,
 };
 
-pub use crate::chunks::vocab::{SimpleVocab, SubwordVocab, Vocab, VocabWrap};
+pub use crate::chunks::norms::NdNorms;
+
+pub use crate::chunks::vocab::{
+    BucketSubwordVocab, ExplicitSubwordVocab, FastTextSubwordVocab, SimpleVocab, Vocab, VocabWrap,
+    WordIndex,
```

Comment:
Is `WordIndex` needed?

Reply:
But a `prelude` should not export every possible symbol. Normal usage is to load an embedding matrix and to look up an embedding. The symbols for doing that should be in the `prelude`. In that sense, the `prelude` is already bloated. It's find if people have to import specific modules for more fine-grained functionality.

(And yes, all the specific vocabs/storages should probably not be in the prelude either, only: the traits and the wrappers (since people will usually want to read as `Embeddings<VocabWrap, StorageWrap>` or `Embeddings<VocabWrap, StorageViewWrap>`).

Reply:
I usually end up importing from between 3 to 6 modules - in addition to prelude - when using `finalfusion` as a dependency, the motivation here was to cut down on that. Maybe we can flatten the structure and re-export symbols in `chunks` from their respective submodules.